bq-connector/docai_bq_connector/doc_ai_processing/Processor.py

```diff
@@ -136,7 +136,7 @@ def _process_sync(
         results_json = documentai.types.Document.to_json(results.document)
```

Comment:
If you need this as a dictionary, there's a built-in function for it

`documentai.Document.to_dict()`